packages/react-components/src/molecules/__tests__/MembersList.test.tsx

```diff
@@ -65,9 +60,9 @@ it('renders a team link for each team provided', async () => {
       members={[
         {
           ...createUserResponse(),
-          displayName: 'Bat Man',
-          role: 'Boss',
-          teams: [
+          firstLine: 'Bat Man',
+          secondLine: 'Boss',
+          thirdLine: [
```

Comment:
I'd add a test to cover that MembersList supports undefined values for `secondLine` and `thirdLine`. What do you think?